cli/cli.go

```diff
@@ -138,6 +138,13 @@ func Main() {
 	deletePitrCmd.Flag("all", "Delete all chunks").Short('a').BoolVar(&deletePitr.all)
 	deletePitrCmd.Flag("force", "Force. Don't ask confirmation").Short('f').BoolVar(&deletePitr.force)
 
+	cleanupCmd := pbmCmd.Command("cleanup", "Delete Backups and PITR chunks")
+	cleanupOpts := cleanupOptions{}
+	cleanupCmd.Flag("older-than", fmt.Sprintf("Delete older than date/time in format %s or %s", datetimeFormat, dateFormat)).StringVar(&cleanupOpts.olderThan)
+	cleanupCmd.Flag("yes", "Don't ask confirmation").Short('y').BoolVar(&cleanupOpts.yes)
```

Comment:
`pbm delete` has `-f/--force` flag for that reason. I agree that `-y/--yes` is more appropriate flag here. But better we have some consistency across UI. How about adding `-y/--yes` flag to the `pbm delete`?
/cc @janwieremjewicz

Reply:
I'm all in for consistency across UI. 
Could we then introduce `-y/--yes` to `pbm delete` and mark `-f/--force` as deprecated (still available)?